### PR TITLE
chore: remove unused dependencies

### PR DESCRIPTION
**Pull request by tvolk131**

### CODE REVIEW

File: fedimint-client/Cargo.toml [hunk 1]
```diff
@@ -25,7 +25,6 @@ path = "src/lib.rs"
 
 [dependencies]
 anyhow = { workspace = true }
-aquamarine = { workspace = true }
```
Comment (m1sterc001guy) [inline fedimint-client/Cargo.toml:28]: This is still needed (its why your build is failing) but otherwise great cleanup!

Comment (tvolk131) [reply fedimint-client/Cargo.toml]: Done! I was running `just check` and `just check-wasm` locally but forgot to run `just build-docs`.

File: fedimint-core/Cargo.toml [hunk 2]
```diff
@@ -60,15 +60,12 @@ url = { workspace = true, features = ["serde"] }
 
 [dev-dependencies]
 test-log = { workspace = true }
-tokio-test = "0.4.4"
```
Comment (oleonardolima) [inline fedimint-core/Cargo.toml:63]: I guess this one's still needed, check the CI failing step.

**Review by oleonardolima — COMMENTED** [review]
Did you use any specific tool to find out the duplicated ones ?

I wasn't able to find it with udeps in the past (might've used wrongly)

**Comment by dpc** [comment]
How is `cargo udeps` not detecting these?

**Comment by tvolk131** [comment]
> Did you use any specific tool to find out the duplicated ones ?

Nope 😬 just happened to notice a few that weren't needed and decided to go through all `Cargo.toml` files by hand

> How is `cargo udeps` not detecting these?

... ¯\_(ツ)_/¯

File: fedimint-core/Cargo.toml (hunk 2 above)
Comment (tvolk131) [reply fedimint-core/Cargo.toml:63]: Fixed!

**Review by dpc — APPROVED** [review]
I'm somewhat amazed. :+1:

**Comment by elsirion** [comment]
Is it possible that `udeps` doesn't detect them because ultimately they are still being pulled in by other deps? At least I can't see a net reduction in dependencies in `Cargo.lock`.

**Pull request merged**